src/chains.ts

```diff
@@ -19,68 +21,69 @@ const baseL2Addresses = {
   fallbackHandlerAddress: "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
   createCallAddress: "0xB19D6FFc2182150F8Eb585b79D4ABcd7C5640A9d",
   signMessageLibAddress: "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
+  safeSingletonAddress: "0x",
+  simulateTxAccessorAddress: "0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da",
 };
 
-// sepolia is the same as base.
-const sepoliaAddresses = baseL2Addresses;
-
 export const contractNetworks: ContractNetworksConfig = {
-  // ZORA goerli
-  [999]: defaultL2Addresses,
-  // ZORA sepolia testnet
-  [999999999]: defaultL2Addresses,
-  // ZORA mainnet
-  [7777777]: defaultL2Addresses,
-  // base goerli
-  [84531]: baseL2Addresses,
-  // base mainnet
-  [8453]: baseL2Addresses,
-  // pgn sepolia
-  [58008]: defaultL2Addresses,
-  // pgn mainnet
-  [424]: defaultL2Addresses,
+  // // ZORA mainnet
+  // [`${chains.zora.id}`]: defaultL2Addresses,
+  // arbitrium sepolia
+  // [`${chains.arbitrumSepolia.id}`]: baseL2Addresses,
   // sepolia testnet
-  [11155111]: sepoliaAddresses,
+  // [`${chains.sepolia.id}`]: sepoliaAddresses,
 };
 
 const pgn = {
   id: 424 as const,
-  name: 'PGN',
-  network: 'pgn',
+  name: "PGN",
+  network: "pgn",
   nativeCurrency: {
     decimals: 18,
-    name: 'Ether',
-    symbol: 'ETH',
+    name: "Ether",
+    symbol: "ETH",
   },
   rpcUrls: {
     default: {
-      http: ['https://rpc.publicgoods.network'],
-      webSocket: ['wss://rpc.publicgoods.network'],
+      http: ["https://rpc.publicgoods.network"],
+      webSocket: ["wss://rpc.publicgoods.network"],
     },
     public: {
-      http: ['https://rpc.publicgoods.network'],
-      webSocket: ['wss://rpc.publicgoods.network'],
+      http: ["https://rpc.publicgoods.network"],
+      webSocket: ["wss://rpc.publicgoods.network"],
     },
   },
   blockExplorers: {
-    etherscan: { name: 'Explorer', url: 'https://explorer.publicgoods.network' },
-    default: { name: 'Explorer', url: 'https://explorer.publicgoods.network' },
+    etherscan: {
+      name: "Explorer",
+      url: "https://explorer.publicgoods.network",
+    },
+    default: { name: "Explorer", url: "https://explorer.publicgoods.network" },
   },
-}
+};
 
-export const allowedNetworks: { [chainId: number]: chains.Chain }= {
-  [999]: chains.zoraTestnet,
-  [999999999]: chains.zoraSepolia,
-  [424]: pgn,
+export const allowedNetworks: { [chainId: number]: chains.Chain } = {
+  [chains.zoraTestnet.id]: chains.zoraTestnet,
+  [chains.zoraSepolia.id]: chains.zoraSepolia,
+  [chains.arbitrumGoerli.id]: chains.arbitrumGoerli,
+  [chains.arbitrumSepolia.id]: chains.arbitrumSepolia,
+  [chains.arbitrumNova.id]: chains.arbitrumNova,
```

Comment:
zora mainnet is missing - can you please add it back in?

Reply:
Resolved in stack.